api/docs/installation/localvir.md

```diff
@@ -52,6 +52,24 @@ $ source badgeyay/bin/activate
 ```sh
 (badgeyay)$ pip install -r requirements.txt
 ```
+* Create the database. For that we first open the psql shell. Go to the directory where your postgres file is stored.
```

Comment:
Also, mention adding the credentials in [Config file](https://github.com/fossasia/badgeyay/blob/development/api/config/config.py) after setting up the database and change the env variable to local.